README.md

```diff
@@ -1,8 +1,14 @@
-# depject
+# depject 
 
-minimal dependency injection
+> simplest dependency injection
```

Comment:
This came from the package json. They match now but can change them back if anyone has an opinion.